libs/dh/shared/assets/src/assets/b2c-custom-layouts/mitid-unified.html

```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <!doctype html>
-<html>
+<html lang="en">
```

Comment:
nitpick: `lang` is added automatically by Azure

```suggestion
<html lang="en">
```